armi/reactor/converters/axialExpansionChanger.py

```diff
@@ -559,12 +560,13 @@ def _determineLinked(componentA, componentB):
 class ExpansionData:
     """Object containing data needed for axial expansion."""
 
-    def __init__(self, a, setFuel):
+    def __init__(self, a, setFuel: bool, coldHeightsToHot: bool):
```

Comment:
doc string please and describe when coldHeightsToHot would be true and the associated setting.

Also call out that it significantly doesn't preserve mass since component is hot radial dims but heights are cold input dims

Reply:
https://github.com/terrapower/armi/pull/1342/commits/73c16ed9d0a14ffbf3090e619426e6c740fc38b4

> Also call out that it significantly doesn't preserve mass since component is hot radial dims but heights are cold input dims

I would prefer to add this to the docs in some capacity. #935

Reply:
add in doc string too. its a big feature difference than what someone might assume. we dont want someone mis-using in a new case because the doc string doesn't call out the major pitfal

Reply:
https://github.com/terrapower/armi/pull/1342/commits/8f969cbd1bf22e10de01ad1c0fcd98e331b50d73